src/firebolt/common/token_storage.py

```diff
@@ -114,6 +117,7 @@ def __init__(self, salt: str, username: str, password: str):
             salt=b64decode(salt),
             length=32,
             iterations=39000,
+            backend=ossl_backend,
```

Comment:
Should we be using `default_backend()` function instead? At least that's what cryptography looks to suggest https://cryptography.io/en/3.1/hazmat/backends/#cryptography.hazmat.backends.default_backend

Reply:
Yeah, I think we can do this. I took an open ssl backen from current version implementation
https://github.com/pyca/cryptography/blob/36.0.x/src/cryptography/hazmat/primitives/kdf/pbkdf2.py#L52